tests/trx_generator/main.cpp

```diff
@@ -139,18 +124,17 @@ void update_resign_transaction(signed_transaction& trx, fc::crypto::private_key
    }
 }
 
-void push_transactions(p2p_trx_provider& provider, const vector<signed_transaction_w_signer>& trxs, uint64_t& nonce_prefix, uint64_t& nonce, const fc::microseconds& trx_expiration, const chain_id_type& chain_id, const block_id_type& last_irr_block_id)
-{
-      std::vector<signed_transaction> single_send = std::vector<signed_transaction>();
-      single_send.reserve(1);
-
-      for(signed_transaction_w_signer trx: trxs) {
-         update_resign_transaction(trx._trx, trx._signer, ++nonce_prefix, nonce, trx_expiration, chain_id, last_irr_block_id);
-         single_send.emplace_back(trx._trx);
-         provider.send(single_send);
-         single_send.clear();
-         ++_txcount;
-      }
+void push_transactions(p2p_trx_provider& provider, const vector<signed_transaction_w_signer>& trxs, uint64_t& nonce_prefix, uint64_t& nonce, const fc::microseconds& trx_expiration, const chain_id_type& chain_id, const block_id_type& last_irr_block_id) {
+   std::vector<signed_transaction> single_send = std::vector<signed_transaction>();
+   single_send.reserve(1);
```

Comment:
I can add a single send method for you to pass in a single signed txn rather than have to create this vector.

Reply:
Sounds good. We can earmark that and circle back once it is in the provider code.